project/.travis/before_install_test.sh.twig

```diff
@@ -5,9 +5,16 @@ if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then
     mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" /tmp
     echo "memory_limit=3072M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 
+    {% if 'mongodb' in services %}
     if [ ${TRAVIS_PHP_VERSION} '<' '7.0' ]; then
         echo "extension=mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+    else
+        echo "extension=mongodb.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+
+        # Backwards compatibility with old mongo extension
+        composer require "alcaeus/mongo-php-adapter" --no-update
```

Comment:
@Soullivaneuh this lib is only required if you use php7. Not sure what happens if you don't but it makes sense. Can you please resolve the conflicts, merge and deploy? I'm afraid @jordisala1991 is unavailable ATM and as you know, this is a blocker the next major release of exporter.